protocol/x/clob/keeper/equity_tier_limit.go

```diff
@@ -149,8 +145,10 @@ func (k Keeper) ValidateSubaccountEquityTierLimitForNewOrder(ctx sdk.Context, or
 	if lib.MustConvertIntegerToUint32(equityTierCount) >= equityTierLimit.Limit {
 		return errorsmod.Wrapf(
 			types.ErrOrderWouldExceedMaxOpenOrdersEquityTierLimit,
-			"Opening order would exceed equity tier limit of %d. Order id: %+v",
+			"Opening order would exceed equity tier limit of %d. Order count: %d, total net collateral: %+v, order id: %+v",
 			equityTierLimit.Limit,
+			equityTierCount,
+			netCollateral,
```

Comment:
Do you think we should include the block height in the error as well? Up to you.

Reply:
That seems like information that should be added to all CheckTx errors outside of just this one and would make sense for it to be added elsewhere.